pkg/test/resourcefixture/testdata/basic/run/v1beta1/runjob/basicjob/_http.log

```diff
@@ -243,31 +246,22 @@ X-Xss-Protection: 0
 
 ---
 
-PATCH https://run.googleapis.com/v2/projects/${projectId}/locations/us-central1/jobs/${jobID}?alt=json
+PATCH https://run.googleapis.com/v2/projects/${projectId}/locations/us-central1/jobs/${jobID}?%24alt=json%3Benum-encoding%3Dint
 Content-Type: application/json
 User-Agent: kcc/${kccVersion} (+https://github.com/GoogleCloudPlatform/k8s-config-connector) kcc/controller-manager/${kccVersion}
+X-Goog-Request-Params: location=us-central1
 
 {
-  "launchStage": "GA",
+  "launchStage": 4,
+  "name": "projects/${projectId}/locations/us-central1/jobs/runjob-${uniqueId}",
   "template": {
-    "taskCount": 1,
     "template": {
       "containers": [
         {
           "image": "us-docker.pkg.dev/cloudrun/container/hello",
-          "name": "container-name",
-          "resources": {
```

Comment:
TF-based reconciler sends a fixed default value for container in the HTTP request (this is also the default value GCP gives if not specified)